DeAceroDLabs/marketplace-frontend
# [DMP-738] Real data offers payment

## What is done

[[DMP-738]  Real data offers payment](https://deacero.atlassian.net/browse/DMP-738)

## How is done

- Used SelectedOfferContext on payment view.
- Refactor to paymentItem.
- Added test for offers.
- This part now is also covered:
![image](https://user-images.githubusercontent.com/81311646/209734982-83411786-7629-43fb-b40a-7d363ecfd5c7.png)


## How to test

1. Run the project via `npm start`.
2. Click on any product.
3. Click `Agregar al carrito` button.
4. Select an offer.
5. Go to payment view. You should see the real offer data on step 2.

## Evidence
![image](https://user-images.githubusercontent.com/81311646/209734913-df76caa1-7861-44cd-aa71-cb17af587140.png)

## Doubts

I think the name is too long but it is the one that is received: 
![image](https://user-images.githubusercontent.com/81311646/209734953-0e20b6f9-505c-4a32-98d0-c2f4c0cf95d2.png)
